### PR TITLE
[TW-954] Doc customizable Oauth

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good to me!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Review by giridharvc7 — CHANGES_REQUESTED** [review]

File: src/pages/docs/sending-requests/authorization.md [hunk 1]
```diff
@@ -323,14 +318,15 @@ On the **Configuration Options** tab:
 * **State** - An opaque value to prevent cross-site request forgery.
 * **Client Authentication** - Send a Basic Auth request in the header, or client credentials in the request body. After upgrading to a new version, change the value here to avoid problems with client authentication.
 
-On the **Advanced Options** tab:
+Select **Advanced Options** to customize the token:
 
-* **Resource** - A URI that indicates the resource or target service where the token is intended to be used.
-* **Audience** - A URI that indicates the target audience or service where the token is intended to be used.
+* **Refresh Token URL** - The endpoint for the refresh server, to exchange the refresh token for an access token. If empty, the refresh token URL will be the same as the access token URL.
+* **Auth Request**, **Token Request**, and **Refresh Request** - Define any custom parameters you want to send with auth requests, token requests, or refresh requests. For each key-value pair you define, select an option to send the parameter in the request body, request URL, or request headers.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/authorization.md:325]: I added the following:
> If you add multiple keys with same key name, they will be sent with the request as an array.

File: src/pages/docs/sending-requests/authorization.md [hunk 2]
```diff
@@ -323,14 +318,15 @@ On the **Configuration Options** tab:
...
+    * If you added resource or audience URIs in a previous version of Postman, you can import them as custom parameters. Select the **Audience and Resource** tab to review the resources and audiences, then select **Move Audience and Resource**. For all grant types except implicit, the parameters are added to the token request. For implicit grant types, the parameters are added to the auth request.
 
-When your config is complete, select __Get New Access Token__.
+When the configuration is complete, select __Get New Access Token__.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/authorization.md:328]: I added the following:
> The configuration information is saved along with the access token and will be used every time the token is refreshed.

**Review by giridharvc7 — APPROVED** [review]

**Pull request merged**